### ISSUE TITLE
Ok to use library even though labelled as experimental?

### ISSUE THREAD

**Issue by patfelst** [RobTillaart/SGP30#7]
Hi Rob, as per the issue title, I'm about to start a new project with this sensor, just wondering if I should use your library or Adafruit's? I don't need the "experimental support for H2 and Ethanol concentration". So are all the other functions ok to use?

Thanks, Patrick

**Comment by RobTillaart** [comment]
Hi Patrick,
From functional point of view both libraries will do their job, otherwise you have the sources and are able to patch or to file an issue. For both libraries you should verify the correct working within your project, as you might use it in a way not tested or foreseen by library makers.

Adafruits libraries (1500+?) are good and well maintained. There is a company supporting them, and they have regular updates.
My libraries are mainly maintained by me. Normally i pick up issues in a day or two. But sometimes it takes longer.

There is a difference in license. If your project is a commercial product, you should compare them too.

So in short, yes please use the library and if you find (missing) issues let me know.

**Comment by patfelst** [comment]
Thanks Rob. ok I guess I really meant is the whole library considered experimental, but from a quick browse of the code it looks like its just the calculated H2 and Ethanol part. Yes I always try and verify libraries are working correctly.

My work isn't commercial, well not yet :) but I should remember to read and include licenses.

I appreciated your response time with the circular buffer RunningAverage. I will give this one a try and report any issues I find.

**Comment by RobTillaart** [comment]
Whole life is experimental 😀
More serious, most of the code should work as intended so it is indeed especially the H2 & Eth . part that is experimental as I have no means to test those in a good, scientific correct and safe way.

**Comment by patfelst** [comment]
I'll close this issue, and raise anything specific in a new one if needed. Thanks again.
